deploy/template.yaml

```diff
@@ -1309,4 +1479,4 @@ Outputs:
     Value: !GetAtt ApiGateway.ApiEndpoint
   CloudFrontDistributionEndpoint:
     Description: The Application URL.
-    Value: !FindInMap [EnvironmentVariables, !Ref Environment, BASEURL ]
+    Value: !FindInMap [EnvironmentVariables, !Ref Environment, BASEURL ]
```

Comment:
:) Could I be a real pain and ask for a newline at the end of this file?

Reply:
Sure :)